src/Features/Core/Portable/Diagnostics/DiagnosticResultSerializer.cs

```diff
@@ -167,6 +167,7 @@ private static void Serialize(ObjectWriter writer, AnalyzerTelemetryInfo telemet
             writer.WriteInt32(telemetryInfo.OperationBlockStartActionsCount);
             writer.WriteInt32(telemetryInfo.OperationBlockEndActionsCount);
             writer.WriteInt64(telemetryInfo.ExecutionTime.Ticks);
+            writer.WriteBoolean(telemetryInfo.Concurrent);
```

Comment:
❓ Is there a version number for this somewhere that needs to be updated so it doesn't try to deserialize data in the old format?

Reply:
:memo: I'm specifically wondering about any changes necessary in the following locations:

https://github.com/dotnet/roslyn/blob/5092197916497c0976084ce1e06bae37b7dfc71a/src/Features/Core/Portable/Diagnostics/DiagnosticResultSerializer.cs#L22

https://github.com/dotnet/roslyn/blob/5092197916497c0976084ce1e06bae37b7dfc71a/src/Workspaces/Core/Portable/Diagnostics/DiagnosticDataSerializer.cs#L24-L31

Reply:
@heejaechang This looks like a question for you :arrow_up:

Reply:
DiagnosticresultSerializer doesn't support persisting its stream. so no versioning is needed. it is only for moving data between VS and OOP live. and we don't support mixing VS with different version of OOP.

simply put, no change needed